lib/cap/authors_poller.rb

```diff
@@ -212,19 +204,20 @@ def process_record_for_new_author(cap_profile_id, record)
           logger.warn "New author has authorship which will be skipped; Author.find_by(cap_profile_id: #{cap_profile_id})"
           @new_auth_with_contribs += 1
         end
-        queue_author_for_harvest(author)
+        queue_author_for_harvest(author,new_record: true)
         author.save!
         @new_author_count += 1
       end
 
       # Add author to job queue if it's harvestable and new/changed
       # @param [String] `skip_message` logs this message if it skips adding author to queue
-      def queue_author_for_harvest(author)
-        if (author.new_record? || author.changed?) && author.harvestable?
-          @new_or_changed_authors_to_harvest_queue << author.id
+      def queue_author_for_harvest(author,options)
+        if author.harvestable?
+          @new_authors_to_harvest_queue << author.id if options[:new_record]
+          @changed_authors_to_harvest_queue << author.id if Settings.CAP.HARVEST_ON_CHANGE && !options[:new_record] && author.changed?
         else
-          @no_sw_harvest_count += 1
-          logger.info "Author marked as not harvestable or did not change. Skipping Author.find_by(cap_profile_id: #{author.cap_profile_id})"
```

Comment:
seems overkill to log the fact that we didn't harvest because something did *not* change.  In fact, probably overkill even to log the fact that nothing was harvested